README.md

```diff
@@ -124,7 +124,7 @@ The tests can be run via Pytest while SSH'd as root into a master node of the cl
     This test user has a known login token with far future expiration. DO NOT USE IN PRODUCTION.
     After the test, remember to delete the test user.
 
-    For more information, see [User Management](https://dcos.io/docs/latest/administration/user-management/).
+    For more information, see [User Management](https://docs.mesosphere.com/1.12/security/oss/user-management/).
```

Comment:
I think, we should replace this link with 
`https://docs.mesosphere.com/latest/security/oss/user-management/` 

Note the the `latest` that gets redirected to 1.12 at the moment, and will hopefully be redirected to the supported version in the future.